src/2_RTB/NormeDiProgetto/content.typ

```diff
@@ -0,0 +1,68 @@
+#import "functions.typ": glossary, team
+
+/*************************************/
+/*    INSERIRE SOTTO IL CONTENUTO    */ 
+/*************************************/
+
+= Introduzione
+
+== Glossario
+Al fine di evitare possibili ambiguità relative al linguaggio utilizzato nei documenti, viene
+fornito il _Glossario_, nel quale sono presenti tutte le definizioni di termini aventi uno
+specifico significato che vuole essere disambiguato. Tali termini, sono scritti in _corsivo_ e
+marcati con una #sub("G") a pedice.
+
+== Scopo del Documento
+Il presente documento ha come scopo la definizione delle #glossary[best practices] e del #glossary[way of working] che ogni componente del #team ha l'obbligo di rispettare durante l'intero svolgimento del progetto. In questo modo si prova a garantire un metodo di lavoro omogeneo, verificabile e miglorabile nel tempo.
+
+#pagebreak()
+
+= Processi di Supporto
+
+== Documentazione
+
+=== Scopo
+
+Lo scopo del seguente processo è la definizione degli standard e degli strumenti utili alla stesura di tutti i documenti, necessari alla buona riuscita del progetto.
+
+=== Ciclo di Vita del Documento
+/ #glossary[Corsie di Stato]: descrivono lo stato attuale del documento.
+/ Backlog: attività individuate da svolgere.
+/ Ready: attività individuate per il completamento durante il prossimo #glossary[sprint].
+/ In Progress: attività che sono in corso d'opera da parte dei redattori.
+/ Ready to Review: attività svolte che sono pronte per essere revisionate.
+/ In Review: attività in corso di revisione da parte dei revisori.
+/ Done: attività le cui modifiche sono state revisionate e accettate.
+
+Ogni nuovo documento segue le fasi del seguente #glossary[workflow]:
++ Si crea un branch per lo sviluppo del nuovo documento e si mette in uso.
```

Comment:
Toglierei la parola "nuovo", il passo 1 è compreso nella procedura di modifica in cui il documento non è nuovo.